app/src/main/res/values/strings.xml

```diff
@@ -73,4 +73,6 @@
     <string name="OWM_API_KEY">OWM API Key</string>
     <string name="apply">Apply</string>
     <string name="cancel">Cancel</string>
+    <string name="location_disabled_message">In order to find watches the location needs to be enabled. Once paired you can safely disable it again.</string>
```

Comment:
I would rephrase it a tiny bit: "... the location service has to be enabled."
You are not enabling a location. You are enabling a services thats all about the location.
Other than that I think we should get that merged asap, wait for a few translations and then push out a new release.